api/src/functions/express/app.ts

```diff
@@ -326,7 +326,7 @@ app.use(
       ["dynamics/housing", dynamicsHousingHealthCheckClient],
       ["dynamics/license-status", dynamicsLicenseHealthCheckClient],
       ["webservice/license-status", webServiceLicenseStatusHealthCheckClient],
-      ["webservice/apiFormationHealthCheck", apiFormationHealthCheckClient],
+      ["webservice/formation-health", apiFormationHealthCheckClient],
```

Comment:
[sand/pebble] Is there a better name for `apiFormationHealthCheckClient` that better fits the patterns of the variables above?

Reply:
done